config/resource-manager.hcl

```diff
@@ -25,8 +25,8 @@ resource_manager "iothub" "2022-04-30-preview" {
   readme_file_path = "../../swagger/specification/iothub/resource-manager/readme.md"
 }
 
-resource_manager "network" "2022-05-01" {
-  swagger_tag = "package-2022-05"
+resource_manager "network" "2022-07-01" {
+  swagger_tag = "package-2022-07"
   readme_file_path = "../../swagger/specification/network/resource-manager/readme.md"
 }
```

Comment:
we need to introduce a new version for this, can you duplicate the lines so we generate both 2022-05 and 2022-07?

Reply:
thanks for your time, the config has been updated.